docs/src/modeling/expressions/functions/README.md

```diff
@@ -57,4 +59,15 @@ gespeichert wird.
 const tFileCoos = JSON.parse(execution.getVariable('sachakten'));
 const tFirstCoo = tFileCoos[0];
 execution.setVariable('firstCoo', tFirstCoo);
+```
+
+Um zwei Listen zusammenzuführen und sie wiederum an eine Prozessvariable zu übergeben, muss die zusammengeführte Liste 
+mit `S()` serialisiert werden. Folgendes Beispiel zeigt die Zusammenführung von zwei Listen.
+
+```
+const list_1 = JSON.parse(execution.getVariable('list_1'));
+const list_2 = JSON.parse(execution.getVariable('list_2'));
+const listCon = list_1.concat(list_2);
+const listConString = JSON.stringify(listCon);
```

Comment:
Das Stringify erscheint mir auf den ersten Blick nicht notwendig, wenn die Daten bereits im richtigen Format im Skript sind. 

Ich müsste das aber nochmal nachstellen für den Use-Case. Hängt aber von dem anderen Kommentar ab.
https://github.com/it-at-m/digiwf-core/pull/1692/files#r1608051631

Reply:
Versuch mal direkt auf dem `JSON-Node` Objekt von Spin zu arbeiten, so wie es als Variable vorliegt.

Eventuell kannst du direkt auf der [SpinList](https://javadoc.io/doc/org.camunda.spin/camunda-spin-core/latest/org/camunda/spin/SpinList.html) zugreifen und die `addAll()` Methode nutzen, sodass du nicht parsen musst

Reply:
Damit man auf die SpinList zugreifen kann muss man das Json mit `S()` aufrufen und darauf `elements(). Dànn geht sogar `addAll()`
```
const coos_1 = execution.getVariable('coos_1');
const coos_2 = execution.getVariable('coos_2');

const coosArray_1 = S(coos_1).elements(); //SpinList
const coosArray_2 = S(coos_2).elements(); //SpinList

coosArray_1.addAll(coosArray_2);

//execution.setVariable('contentCoos', coosArray_1.toString());  //["COO.2150.9151.1.1233734", "COO.2150.9151.1.1233730"] Type: String
execution.setVariable('contentCoos', S(coosArray_1.toString()));  //["COO.2150.9151.1.1233734", "COO.2150.9151.1.1233730"] Type: Json
```
Allerdings muss wieder mit `toString()` und `S()` an die Execution-Variable übergeben werden, damit sie im richtigen Format vorliegt.

Reply:
> damit sie im richtigen Format vorliegt.

Als String oder JSON Type?

Reply:
Das ist Json.

Leider muss man dieses `S(...toString())` machen. Ich finde keine schöne Variante, wie man aus einer SpinList ein SpinJsonNode macht, das ein pures Json Array ist.